test/atm/global-workflow/jjob_run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+bindir=$1
+srcdir=$2
+
+# run jjob
+export HOMEgfs=$srcdir/../../ # TODO: HOMEgfs had to be hard-coded in config
+echo $HOMEgfs
+
+export EXPDIR=$bindir/test/atm/global-workflow/testrun/experiments/gdas_test
+export PDY=20210323
+export cyc=18
+export CDATE=${PDY}${cyc}
+export ROTDIR=$bindir/test/atm/global-workflow/testrun/ROTDIRS
+export CDUMP=gdas
+export DATAROOT=$bindir/test/atm/global-workflow/testrun/RUNDIR
+export COMIN_GES=${bindir}/test/atm/bkg
+export pid=${pid:-$$}
+export jobid=$pid
+export COMROOT=$DATAROOT
+export APRUN_ATMANAL="$MPIEXEC_EXEC $MPIEXEC_NPROC 6"
```

Comment:
I'm OK with this, but `APRUN_ATMANAL` is also exported after sourcing the configs.

Reply:
No need to set `APRUN_ATMANAL` twice.  Remove from `jjob_run.sh`.